components/script/dom/node.rs

```diff
@@ -2476,20 +2487,24 @@ impl Node {
                 Some(index)
             }
         };
-        // Step 6. pre-removing steps for node iterators
-        // Step 7.
+
+        // TODO: Step 4. Pre-removing steps for node iterators
+
+        // Step 5.
         let old_previous_sibling = node.GetPreviousSibling();
-        // Step 8.
+
+        // Step 6.
         let old_next_sibling = node.GetNextSibling();
-        // Steps 9-10 are handled in unbind_from_tree.
+
+        // Step 7.
```

Comment:
Specification text for text numbers in HTML seems to change pretty frequently, so please include specification text with the number in inline comments.